src/locales/en/en.json

```diff
@@ -48,28 +48,28 @@
   },
   "generateseedphrase": {
     "onboarding": {
-      "title": "Generate your Seed Phrase",
+      "title": "Generate seed phrase",
       "paragraph": {
-        "top": "It is critical that you securely store this secret recovery phrase, also referred to as a mnemonic, seed, or backup phrase. You will require this phrase in the event that you change mobile devices or need to regain access to your identity wallet profile.",
-        "bottom": "This phrase is essentially a representation of your Identity Wallet profile private key. This means that anyone with access to it can control your identity. We strongly recommend not using screenshots or storing this phrase without encryption."
+        "top": "Think of your secret recovery phrase as a safety net for your identity. If you ever lose your phone or switch to a new wallet, this phrase will help you recover your identity.",
+        "bottom": "It's important to keep these words safe and sound! Store them in a secure location, like a password manager, and remember to never share them with anyone."
       },
       "button": {
-        "continue": "Continue"
+        "continue": "Generate"
       }
     },
     "segment": "{{length}} words",
     "privacy": {
       "overlay": {
-        "text": "Select the 'View Seed Phrase' when you are ready. Remember to ensure that other people cannot also view your screen.",
+        "text": "Press the ‘view’ button when you’re ready to see your seed phrase. Remember to make sure nobody is looking!",
         "button": "View Seed Phrase"
       }
     },
     "alert": {
       "confirm": {
-        "text": "Take a moment now to confirm you have stored your seed phrase accurately and in the correct order.",
+        "text": "Have you double-checked you’ve copied your 16 word phrase correctly?",
```

Comment:
It's now 18 words - we had to make some changes when we implemented this in the core as we were missing something